src/platform/screen.c

```diff
@@ -289,6 +289,23 @@ void platform_screen_center_window(void)
     window_pos.centered = 1;
 }
 
+#ifdef _WIN32
+void platform_screen_recreate_texture(void)
+{
+    // On Windows, if ctrl + alt + del is pressed during fullscreen, the rendering context may be lost for a few frames
+    // after restoring the window, preventing the texture from being recreated. This forces an attempt to recreate the
+    // texture every frame to bypass that issue.
+    if (!SDL.texture && SDL.renderer && setting_fullscreen()) {
+        SDL_DisplayMode mode;
+        SDL_GetWindowDisplayMode(SDL.window, &mode);
+        system_resize(scale_pixels_to_logical(mode.w), scale_pixels_to_logical(mode.h));
```

Comment:
`system_resize` causes an SDL resize event which is handled in the next run of the event loop. Do we really need that, or is `screen_set_resolution` sufficient?

Reply:
We need at least a `platfofm_screen_resize` to recreate the texture.